app/src/main/java/org/oppia/app/player/state/itemviewmodel/DragAndDropSortInteractionViewModel.kt

```diff
@@ -74,8 +74,7 @@ class DragAndDropSortInteractionViewModel(
   override fun getPendingAnswer(): UserAnswer {
     val userAnswerBuilder = UserAnswer.newBuilder()
     val listItems = choiceItems.map { it.htmlContent }
-    val listItemsHtml = choiceItems.map { it.htmlContent.htmlList.joinToString() }
-    userAnswerBuilder.htmlAnswer = convertSelectedItemsToHtmlString(listItemsHtml)
+    userAnswerBuilder.listOfHtmlAnswers = convertItemsToAnswer(listItems)
```

Comment:
fix the ktlint issues coming in this file

Reply:
done

Reply:
done